tests/docker_unit/Dockerfile

```
@@ -2,7 +2,7 @@
 FROM dela-builder AS builder
 
 # Test environment
-FROM rust:1.93.0-alpine3.21
+FROM dela-builder
```

Hadolint (2.15.1)

[warning] Always tag the version of an image explicitly

(DL3006)